public/pages/techtalk.html

```diff
@@ -51,46 +51,60 @@
     <!-- Image Scrolling Banner v1.0 -->
     <div class="image-banner image-banner-1 full-width-in-main-body">
       <div class="banner-text">
-        <h3>Online Tech Talk: Are Code Bootcamps a fast-track to a career in tech?</h3>
-        <a href="#zoom" class="button" style="max-width: 12em;">Join Meeting</a>
+
+        <h3>Online Tech Talk: Building a Data Analytics Platform  Using Google Cloud</h3>
+        <a href="#zoom" class="button" style="max-width: 20em;">Zoom link will be availabe at 6pm on February 16!</a>
+
+
       </div>
     </div>
 
     <div class="twoColumnGrid">
       <div class="">
-        <h2>Tech Talk: Are Code Bootcamps a fast-track to a career in tech?</h2>
-        <p>
-          Our Wednesday-night lab is online for now due to Covid! Join us February 9th from 6:30-8:00pm via Zoom using the link <a href="#zoom">below</a>!
-        </p>
-        <div style="float: left; margin: 0 10px; padding: 10px;float: left;text-align: center;">
-          <img src="../img/Erin-shaw.jpg" style="max-width; 150px;"/>
-          <h4 style="margin: 0; text-align: center;">Erin Shaw </h4>
-          <p style="margin-top: 5px;">Software Engineer @ Mailchimp<br>Bootcamp Graduate</p>
+          <h2>Tech Talk: Building a Data Analytics Platform  Using Google Cloud</h2>
+          <p>
+            Our Wednesday-night lab is online for now due to Covid! Join us February 16th from 6:30-7:30pm via Zoom using the link <a href="#zoom">below</a>!
+          </p>
+          <div style="float: left; margin: 0 10px; padding: 10px;float: left;text-align: center;">
```

Comment:
Move this Div to the end of the section, below all the `<p>` elements, and delete the `float` rules from the `style` attribute.